modules/bitgo/test/v2/unit/coins/utxo/util/keychains.ts

```diff
@@ -23,6 +31,19 @@ export function toKeychainBase58(k: BIP32Interface): KeychainBase58 {
   };
 }
 
+export function toKeychainObjects(rootWalletKeys: RootWalletKeys, walletPassphrase: string): KeyDoc[] {
+  return rootWalletKeys.triple.map((bip32, keyIdx) => {
+    const pub = bip32.neutered().toBase58();
+    return {
+      id: getSeed(pub).toString('hex'),
+      pub,
+      source: ['user', 'backup', 'bitgo'][keyIdx],
```

Comment:
Is there not already a const for this array? If not, let's please export a const and try to reuse it.